dashboard/README.md

```diff
@@ -40,29 +40,38 @@ dashboards with different configurations.
 [replicator]: cmd/postgres_replicator/README.md
 [replicatorconfig]: config/postgres_replicator/default/config.yaml
 
+## Deploy dashboard with PSM
+
+PSM dashboard is not included in the deployment by default, following command is
```

Comment:
I would not include this section.

Instead, I think we should update both the cloud_build.yaml and the manual build instructions to include the PSM variable in the table of variables, and explain how it works there.